File: ts/project/.sdk/src/cmp/swift/fragment/Entity.fragment.swift
// EntityName entity client for the ProjectName SDK.

import Foundation

public final class EntyClass: ProjectNameEntityBase {
  public init(_ client: ProjectNameSDK, _ entopts: VMap? = nil) {
    super.init(client, entopts, "entityname")
  }

  public override func make() -> Entity {
    return EntyClass(client, cloneOpts())
  }

  // #LoadOp

  // #ListOp

  // #CreateOp

  // #UpdateOp

  // #RemoveOp
}

File: ts/project/.sdk/src/cmp/csharp/fragment/Entity.fragment.cs
// EntityName entity client for the ProjectName SDK.

using Voxgig.Struct;

namespace ProjectNameSdk.Entity;

public class EntyClass : ProjectNameEntityBase
{
    public EntyClass(ProjectNameSDK client, Dictionary<string, object?>? entopts = null)
        : base(client, entopts, "entityname")
    {
    }

    public override IEntity Make()
    {
        return new EntyClass(client, CloneOpts());
    }

    // #LoadOp

    // #ListOp

    // #CreateOp

    // #UpdateOp

    // #RemoveOp
}
